Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -382,7 +384,9 @@ OriginStorageManager& NetworkStorageManager::originStorageManager(const WebCore:
         OriginQuotaManager::IncreaseQuotaFunction increaseQuotaFunction = [sessionID = m_sessionID, origin, connection = m_parentConnection] (auto identifier, auto currentQuota, auto currentUsage, auto requestedIncrease) mutable {
             IPC::Connection::send(connection, Messages::NetworkProcessProxy::IncreaseQuota(sessionID, origin, identifier, currentQuota, currentUsage, requestedIncrease), 0);
         };
-        uint64_t quota = m_defaultOriginQuota;
+        // Use double for mutlipication to preserve precision.
```

Comment:
s/mutlipication/multiplication.
Also we should probably move standardReportedQuota. closer to where it is actually used.

Reply:
standardReportedQuota is on `NetworkStorageManager` because `OriginStorageManager` or `OriginQuotaManager` does not keep track of `WebCore::ClientOrigin`, and thus does not know policy for third-party origin.

Do you mean to move `defaultStandardReportedQuota` to `OriginQuotaManager`?